install/tools/ipa-compat-manage

```diff
@@ -100,7 +100,10 @@ def main():
         if dirman_password is None:
             sys.exit("Directory Manager password required")
 
-    api.bootstrap(context='cli', in_server=True, debug=options.debug)
+    api.bootstrap(context='cli',
+                  in_server=True,
+                  debug=options.debug,
+                  confdir=paths.ETC_IPA)
```

Comment:
It does not matter at all that the script works with IPA_CONFDIR - there is no use case for this and it only introduces additional maintenance burden for us.